src/main/java/ch/astorm/jotlmsg/io/FlatEntryStructure.java

```diff
@@ -0,0 +1,62 @@
+package ch.astorm.jotlmsg.io;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Represents an [MS-OXCDATA] 2.3.2 FlatEntry Structure in Java.
+ * 
+ * @author Guido Stein
+ *
+ */
+public class FlatEntryStructure {
+    private long size; // Number of bytes of following EntryID field.
+    private byte[] entryID;
+
+    /**
+     * Returns the number of bytes of following EntryID field.
+     * 
+     * @return Number of bytes.
+     */
+    public long getSize() {
+        return size;
+    }
+
+    /**
+     * Returns the EntryID bytes.
+     * 
+     * @return Byte array representation of EntryID.
+     */
+    public byte[] getEntryID() {
+        return entryID;
+    }
+
+    /**
+     * Sets the EntryID bytes.
+     * 
+     * @param entryID Byte array data to set.
+     */
+    public void setEntryID(byte[] entryID) {
+        this.entryID = entryID;
+        this.size = entryID.length;
+    }
+
+    /**
+     * Default constructor
+     */
+    public FlatEntryStructure() {
+    };
```

Comment:
This constructor can be entirely removed.

Reply:
I hava tried to remove it but it seems that the generic stuff doesn't work without it.